src/server/config.rs

```diff
@@ -69,6 +69,8 @@ pub struct Config {
     pub end_point_stack_size: ReadableSize,
     pub end_point_recursion_limit: u32,
     pub end_point_batch_row_limit: usize,
+    pub snap_max_write_bytes_per_sec: ReadableSize,
```

Comment:
Please add this configuration and some comments about it to `etc/config-template.toml`.